### ISSUE TITLE
[BUG] Unexpected 'invalid configuration argument' after trying to read one row from an empty ORC file

### ISSUE THREAD

**Issue by rommelDB** [rapidsai/cudf#3001]
**Describe the bug**
After trying to read some rows from an empty ORC file, cudf throw an 'invalid configuration argument' even when trying to read another non-empty file with valid parameters.

**Steps/Code to reproduce bug**
Minimal script:
```
import cudf
data_dir='/path/to/data/'

# (only schema, no data) RuntimeError: cuDF failure at orc_reader_impl.cu:481: No decompressible data found
try:
    df1 = cudf.read_orc(data_dir + 'part-00000.orc', num_rows=1)
except:
    print("Error: 1")

# (file with data OK) RuntimeError: parallel_for failed: invalid configuration argument
df2 = cudf.read_orc(data_dir + 'part-00001.orc', num_rows=1)
```
Test files:
[data-orc.tar.gz](https://github.com/rapidsai/cudf/files/3701374/data-orc.tar.gz)

**Traceback:**
```
$ python orc-bug-cudf.py 
Error: 1
Traceback (most recent call last):
  File "orc-bug-cudf.py", line 11, in <module>
    df2 = cudf.read_orc(data_dir + 'part-00001.orc', num_rows=1)
  File "/home/rommel/miniconda3/envs/blazingsql-nightly-0.10-develop/lib/python3.7/site-packages/cudf/io/orc.py", line 46, in read_orc
    filepath_or_buffer, columns, stripe, skip_rows, num_rows, use_index
  File "cudf/_lib/orc.pyx", line 49, in cudf._lib.orc.read_orc
  File "cudf/_lib/orc.pyx", line 93, in cudf._lib.orc.read_orc
RuntimeError: parallel_for failed: invalid configuration argument
```

**Expected behavior**
It is clear that reading at least one line from an empty file should not be allowed, however, subsequent file reads should not be invalidated.

**Environment overview (please complete the following information)**
 - Environment location: Bare-metal
 - Method of cuDF install: conda
```
conda install -c conda-forge -c rapidsai-nightly/label/cuda10.0 cudf=0.10
```

**Environment details**
```
$ conda list | grep rapids
cmake_setuptools          0.1.3                      py_0    rapidsai-nightly
dask-cudf                 0.10.0a191007         py37_2427    rapidsai-nightly/label/cuda10.0
libcudf                   0.10.0a191007     cuda10.0_2427    rapidsai-nightly/label/cuda10.0
libnvstrings              0.10.0a191007     cuda10.0_2427    rapidsai-nightly/label/cuda10.0
librmm                    0.10.0a191007      cuda10.0_136    rapidsai-nightly/label/cuda10.0
nvstrings                 0.10.0a191007         py37_2427    rapidsai-nightly/label/cuda10.0
rmm                       0.10.0a191007          py37_136    rapidsai-nightly/label/cuda10.0
```

**Comment by rommelDB** [comment]
The proposal fix is on PR #3002 .

**Comment by aucahuasi** [comment]
Hi @rommelDB since the PR is merged, I guess we can close this one right? ;)
